docs/gateway/01_overview.md

```diff
@@ -0,0 +1,77 @@
+# LMS Gateway
+
+_See also:_
+
+ * [API Specification](02_api_spec.md) - Details of the end-points
+ * [Testing tools](03_testing_tools.md) - Testing tools and reference implementation
+
+## Overview
+
+The LMS Gateway allows you to access the main Hypothesis (`h`) API to retrieve 
+annotation and other data on behalf of and related to a certain user. 
+
+It will also provide you with contextual information about the current LMS 
+context in order to help you:
+
+ * Ask for the right things from `h`
+ * Map between LMS and `h` ids and concepts
+
+### The current implementation is a work in progress
+
+Details of the call structure, required fields and approach are likely to 
+change at this time without warning.
+
+### How it works
+
+You can authenticate with the Gateway end-point using a standard LTI basic 
```

Comment:
`LTI 1.1` for our LTI version sensitive audience.

Reply:
Yes. And it also only works for that. I should make that a bigger deal